Hanson.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
     spec.name                      = "Hanson"
-    spec.version                   = "1.0"
+    spec.version                   = "1.0.1"
```

Comment:
Yeah I'm a member of the organisation but I don't think I have write access to the repo